### PR TITLE
Cat 897

### PR DESCRIPTION
**Pull request by LauraBebek**

### CODE REVIEW

**Comment by LauraBebek** [comment]
created new string which has to added at crowdin first

**Comment by robertpainsi** [comment]
Please remove all unrelated commits (CAT-898 and merge) from the PR.
Also the first letter of the [commit message](https://github.com/Catrobat/Catroid/wiki/Setup-working-environment#commit-guidelines) should be upper case (CAT-897 **A**dded...) and there is a typo in desel**e**ct.

**Comment by avihric** [comment]
> created new string which has to added at crowdin first

I don't understand this really. You did not change any string.

File: catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java [hunk 1]
```diff
@@ -327,15 +330,22 @@ public boolean onContextItemSelected(MenuItem item) {
 
 	@Override
 	public void onSpriteChecked() {
+		Log.d("Lausi", "ONSpriteChecked");
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java]: @avihric: Surely there is :wink:

Comment (avihric) [inline catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java]: Erm still this does not explain the comment, as @LauraBebek mentioned Crowdin. In order to change a string at Crowdin, the `strings.xml` has to be modified, which in this case did not happen.

Btw: "Lausi" is not a good tagname :stuck_out_tongue: , more precisely this line should be removed

Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java]: > Btw: "Lausi" is not a good tagname  , more precisely this line should be removed

And now you know why I commented on this line in the first place. :stuck_out_tongue_winking_eye:

Comment (avihric) [inline catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java]: Well done sir :+1:

**Comment by avihric** [comment]
Reopen when issues have been resolved

**Pull request closed without merging**